### ISSUE TITLE
A11y_.NETCore_RecipesAppMyRecipesEdit_EditRecipe_ScreenReader: Voice over is not announcing label information for slider, increase or decrease button.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#15621]
### Description

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NETCore_iOS_May23; #.NET Core; #WCAG4.1.2; #iOS; #iPhone; #Voiceover;

### Environment Details:​
Application Name: Rachel's Recipe
Operating System: iOS (iPhone 6s)
Screen Reader: Voiceover

### Repro Steps:

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till any existing recipe and double tap.
4. "Recipe" Screen will open.
5. Swipe till "Edit" button and double tap.
6. Start Voice over.
7. Swipe till slider, plus/minus button and observe that Voice over is not announcing label information.

### Actual Results:
Voice over is not announcing label information for slider, increase or decrease button.

**Similar issue is also observed in below scenario:
Issue 1: Recipes App My Recipes New_ New Recipe:**

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till "New" button and double tap.
4. "New Recipe" Screen will open.
5. Start Voice over.
6. Swipe till slider, increase or decrease button and observe that Voice over is not announcing label information.

### Expected Results:
Voice over should be announcing "Review" label information for slider, increase or decrease button.

### User Impact:
Screen reader users will not get information about the labels and will be confused and will not get the purpose of the controls.

### Attachment:

https://github.com/dotnet/maui/assets/90901519/1d86d434-99f8-4a6a-af7a-281e35f92664

### Link to public reproduction project repository

Test

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS version 15.7.6

**Comment by rachelkang** [comment]
Same discussions from https://github.com/dotnet/maui/issues/15623, https://github.com/dotnet/maui/issues/15595

**Comment by PoojaNamde** [comment]
#Regressed:08-22-23; #Closed;

Verified the bug in latest application update.
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment
